ui/app/components/toolbar-filters.js

```diff
@@ -1,3 +1,14 @@
+/**
+ * @module ToolbarFilters
+ * `ToolbarActions` components are containers for Toolbar filters and toggles.
+ *
+ * @example
+ * ```js
+ * <ToolbarFilters>Toolbar Filters</ToolbarFilters>
```

Comment:
I think adding a richer example of what the children of `ToolbarFilters` should be would be helpful. Something like:

```
<ToolbarFilters>
      <input
         type="checkbox"
        name="json"
      />
      <label for="json">JSON</label>
    </ToolbarFilters>
```